docs/architecture/architecture.md

```diff
@@ -273,15 +273,18 @@ When handling the OCI `create` command, `cc-runtime` goes through the following
 4. Create and start the virtual machine that will run the container process. The
   VM will run inside the host namespaces created during step 1, and its `systemd`
   instance will spawn the `cc-agent` daemon.
-5. Register the virtual machine with `cc-proxy`.
-6. The `cc-proxy` waits for the agent to signal that it is ready and then returns
+5. Spawn the `cc-proxy` process providing a single argument:
+  `cc-proxy --uri $(uri)`
+   * A UNIX socket URI, which will be used by the `cc-shim` and `cc-agent` processes will use to pass information between them.
+6. Register the virtual machine with `cc-proxy`.
+7. The `cc-proxy` waits for the agent to signal that it is ready and then returns
   a token. This token uniquely identifies a process within a container inside
   the virtual machine.
-7. Spawn the `cc-shim` process providing two arguments:
+8. Spawn the `cc-shim` process providing two arguments:
```

Comment:
Was surprised you did not change these all to (1.) items whilst here @jodh-intel:-)

Reply:
I considered it, but then decided to go for a minimal change "in line" with the current content to minimise the review cycle time ;)